examples/frontend/README.md

```diff
@@ -9,7 +9,9 @@ This will use the port 8000 (for vLLM), 8501 (for the frontend) and port 65432(f
 1.  ```lmcache_server localhost 65432```  
 And wait until it's ready.  
 2. In one terminal,  
-```LMCACHE_CONFIG_FILE=example.yaml CUDA_VISIBLE_DEVICES=0 python3 -m lmcache_vllm.vllm.entrypoints.openai.api_server --model mistralai/Mistral-7B-Instruct-v0.2 --gpu-memory-utilization 0.8 --port 8000```  
+```LMCACHE_CONFIG_FILE=example.yaml CUDA_VISIBLE_DEVICES=0 python3 -m lmcache_vllm.vllm.entrypoints.openai.api_server --model meta-llama/Llama-3.1-8B-Instruct --gpu-memory-utilization 0.8 --port 8000```  
```

Comment:
Can you elaborate why making the change here?

Reply:
We can revert back. This is simply changing mistral to 3.1B for another task